PANDORA/__init__.py

```diff
@@ -7,7 +7,7 @@
 :meta hide-value:
 '''
 
-PANDORA_data = os.path.join(os.path.dirname(PANDORA_path), 'PANDORA_files', 'data')
+PANDORA_data = os.path.join(os.path.dirname(PANDORA_path), 'Databases', 'default')
```

Comment:
Hi @gcroci2 , I have a question.
By defining PANDORA_data like this, PANDORA will always use the default database also, for instance, for blasting the MHC sequence (Modelling_functions.py, line 847). Will this prevent a user from using their own generated database when blasting? If I am not wrong, there is not an option to select the blast db.
Any idea on how to solve this issue? Maybe just redefining PANDORA_data at the beginning of the user's script will be enough?

Reply:
`PANDORA_data` now is a global variable that is used internally in functions and classes, and the structure of the code does not take into account such a possibility for the user. We could either a) make PANDORA_data an attribute of the Database class, and then modify methods and functions accordingly or b) create a config file (.py, .yml, .toml, ...) in which we put the default folder path, that can be modified by the user. The init of the package then is just going to read out this config file, assigning PANDORA_data to the path that finds there.